report_py3o/models/py3o_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class Py3oTemplate(models.Model):
```

Comment:
No, we keep py3o. The py3o project has existed for many years, it's name won't be changed.